docs/_posts/ahmet-mesut/2024-02-21-drug_matcher_en.md

```diff
@@ -0,0 +1,119 @@
+---
+layout: model
+title: Drug Text Matcher
+author: John Snow Labs
+name: drug_matcher
+date: 2024-02-21
+tags: [en, licensed, drug, textmatcher, clinical]
+task: Named Entity Recognition
+language: en
+edition: Healthcare NLP 5.2.1
+spark_version: 3.0
+supported: true
+annotator: TextMatcherInternalModel
+article_header:
+  type: cover
+use_language_switcher: "Python-Scala-Java"
+---
+
+## Description
+
+This model extracts medication entities in clinical notes using rule-based `TextMatcherInternal` annotator.
+
+## Predicted Entities
+
+`DRUG`
+
+{:.btn-box}
+<button class="button button-orange" disabled>Live Demo</button>
+<button class="button button-orange" disabled>Open in Colab</button>
+[Download](https://s3.amazonaws.com/auxdata.johnsnowlabs.com/clinical/models/drug_matcher_en_5.2.1_3.0_1708546884292.zip){:.button.button-orange.button-orange-trans.arr.button-icon.hidden}
+[Copy S3 URI](s3://auxdata.johnsnowlabs.com/clinical/models/drug_matcher_en_5.2.1_3.0_1708546884292.zip){:.button.button-orange.button-orange-trans.button-icon.button-copy-s3}
+
+## How to use
+
+
+
+<div class="tabs-box" markdown="1">
+{% include programmingLanguageSelectScalaPythonNLU.html %}
+	
+```python
+documentAssembler = DocumentAssembler()\
+    .setInputCol("text")\
+    .setOutputCol("document")
+
+tokenizer = Tokenizer()\
+    .setInputCols(["document"])\
+    .setOutputCol("token")
+
+text_matcher = TextMatcherInternalModel.pretrained("drug_matcher","en","clinical/models") \
+    .setInputCols(["document", "token"])\
+    .setOutputCol("matched_text")\
```

Comment:
@ahmet-mesut if the model is able to extract the entities without setting the case sensitivity and delimeter, please remove them from the sample code.